docs/pages/guides/building-for-tv.mdx

```diff
@@ -27,25 +27,29 @@ The necessary changes to the native Android and iOS files are minimal and can be
   - The Xcode project is modified to target tvOS instead of iOS
   - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
 
-## Quick start
-
-The fastest way to generate a new project is described in the [TV Example](https://github.com/expo/examples/tree/master/with-tv) within the Expo examples repository.
-
-## Integration with an existing Expo project
-
-If you have an existing Expo project, the following walkthrough describes the steps required to modify an Expo project for TV.
+## System requirements for TV development
 
-You will need:
-
-- An Expo project using SDK 50 or later.
 - **Android TV**:
+  - [Node.js (LTS)](https://nodejs.org/en/) on macOS or Linux.
+  - Android Studio (Iguana or later).
   - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple silicon, choose the ARM 64 image. Otherwise, choose the Intel x86_64 image).
   - After installing the Android TV system image, create an Android TV emulator using that image (the process is the same as creating an Android phone emulator).
 - **Apple TV**:
+  - [Node.js (LTS)](https://nodejs.org/en/) on macOS.
   - Xcode 15 or later.
   - tvOS SDK 17 or later. (This is not installed automatically with Xcode. You can install it later with `xcodebuild -downloadAllPlatforms`).
 
-> **Info** If you are starting with an SDK 49 project, you will need to [upgrade to SDK 50 or higher](/workflow/upgrading-expo-sdk-walkthrough/) before proceeding through the following steps.
+## Quick start
+
+The fastest way to generate a new project is described in the [TV Example](https://github.com/expo/examples/tree/master/with-tv) within the Expo examples repository:
+
+<Terminal cmd={['$ yarn create expo-app MyTVProject -e with-tv']} />
+
+For SDK 51 and later, you can start with the [TV Router Example](https://github.com/expo/examples/with-router-tv):
+
+<Terminal cmd={['$ yarn create expo-app MyTVProject -e with-router-tv']} />
+
+(This creates a new project that uses [Expo Router](/router/introduction/) for file based navigation, modeled after the [**create-expo-app** default template](/get-started/create-a-project/).)
```

Comment:
```suggestion
This creates a new project that uses [Expo Router](/router/introduction/) for file-based navigation, modeled after the [**create-expo-app** default template](/get-started/create-a-project/).
```

This sounds like something user should be read when using the second command. So let's make it part of the main flow.